app/components/twitterwall/TwitterwallController.js

```diff
@@ -12,13 +8,23 @@ twitModule.config(['$stateProvider', function($stateProvider) {
     });
 }]);
 
-twitModule.controller('TwitterwallController',
-    ['$rootScope', '$sessionStorage', '$scope',
-        function($scope, $rootScope, $sessionStorage) {
 
-            var tc = this;
-            $scope.hashtag = 'fossasia';
 
-        }
-    ]
-);
\ No newline at end of file
+/*created By aayusharora on 17/2/16*/
+twitModule.controller('TwitterwallController',
+    ['$state','$http','$scope',function($state,$http,$scope){
+      $scope.fetched=false;
+        $http({
+  method: 'GET',
+  url: 'http://loklak.org/api/search.json?q=fossasia',
```

Comment:
Should take the keyword from a config.json or something like that. Should not be hardcoded.